main.py

```diff
@@ -618,6 +627,38 @@ def ffmpeg_post_hook(e: BaseException = None):
     await bot_member.edit(nick=new_nick)
 
 
+async def log_track_playback_exception(e: BaseException, track_info: str) -> None:
+    """
+    If an exception occurred while busting, log it to stdout and
+    send a message to the room. Then end the bust without saying
+    goodbye.
+
+    Some info about the track (e.g. title) is included in the message sent
+    to the room to immediately inform admins about which song had issues, and
+    to give them the ability to assess whether the bust should be restarted from
+    that song, or the next one, after concluding that the song cannot be played.
+
+    Args:
+        e: The exception to log.
+        track_info: The title of the song.
+    """
+    global now_playing_msg
```

Comment:
Seems this we can remove this line, as `now_playing_msg` isn't mentioned further in the function